### ISSUE TITLE
Can't unfollow user

### ISSUE THREAD

**Issue by Mcsavvy** [stanvanrooy/instauto#188]
**Describe the bug**
I tried using the helpers module to unfollow a user

**To Reproduce**
1. from instauto import api, helpers, bot
2. client = api.client.ApiClient(username='foo', password='bar')
3. client.log_in()
4. helpers.friendships.follow_user(client, username='foobar')
> True
5. helpers.friendships.unfollow_user(client, username='davido')
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-16-028e87ed3e64> in <module>
----> 1 helpers.friendships.unfollow_user(client, username='davido')

~/.local/lib/python3.9/site-packages/instauto/helpers/friendships.py in unfollow_user(client, user_id, username)                                                    
    123 
    124     obj = Destroy(str(user_id))
--> 125     resp = client.unfollow_user(obj)
    126     return is_resp_ok(resp)
    127 

AttributeError: 'ApiClient' object has no attribute 'unfollow_user'

In [17]: helpers.friendships.unfollow_user(client, username='davido')
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-17-028e87ed3e64> in <module>
----> 1 helpers.friendships.unfollow_user(client, username='foobar')

~/.local/lib/python3.9/site-packages/instauto/helpers/friendships.py in unfollow_user(client, user_id, username)                                                    
    123 
    124     obj = Destroy(str(user_id))
--> 125     resp = client.unfollow_user(obj)
    126     return is_resp_ok(resp)
    127 

AttributeError: 'ApiClient' object has no attribute 'unfollow_user'

```

**Expected behavior**

**Setup**
Instauto version:
Device profile:
Instagram profile:

**Additional context**

**Comment by stanvanrooy** [comment]
https://github.com/stanvanrooy/instauto/blob/c3b26e7b7b119e1699b7213ae8b74d8f939dee64/instauto/helpers/friendships.py#L125 should be `user_unfollow`, I'll fix and release 2.0.2 later today.

**Comment by stanvanrooy** [comment]
Fixed and release in v2.0.2.  Consider giving `instauto` a star if it helps you in any way ;)
